akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorRetentionSpec.scala

```diff
@@ -198,6 +234,7 @@ class EventSourcedBehaviorRetentionSpec
       replyProbe.expectMessage(State(3, Vector(0, 1, 2)))
     }
 
+    // Should this test be moved out of "EventSourcedBehavior with retention" test block?
```

Comment:
Do you agree that we should move this test below (`snapshot via predicate`) out of this section (`EventSourcedBehavior with retention`), into the new section introduced above (`EventSourcedBehavior with predicate`)?

Reply:
Sure